tox.ini

```diff
@@ -18,6 +18,7 @@ ignore_basepython_conflict = true
 [testenv]
 basepython = python3.6
 deps = {[base]testdeps}
+install_command = pip install --pre {opts} {packages}
```

Comment:
If we only need to use `--pre` then you can use: https://tox.readthedocs.io/en/latest/config.html#conf-pip_pre

Reply:
You really are the tox master.